inspectors/view/qml/notation/fretdiagrams/FretDiagramPopup.qml

```diff
@@ -0,0 +1,39 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import "../../common/"
+
+StyledPopup {
+    id: root
+
+    property QtObject model: null
+
+    implicitHeight: contentColumn.implicitHeight + topPadding + bottomPadding
+    height: implicitHeight
+    width: parent.width
+
+    Component.onCompleted: {
+        model.onSelectionChanged.connect(fretCanvas.selectionChanged)
+    }
+
+    Column {
+        id: contentColumn
+
+        width: parent.width
+
+        spacing: 12
+
+        FretCanvas {
+            id: fretCanvas
+
+            visible: model ? model.showFretCanvas() : false
+
+            model: root.model
+
+            signal selectionChanged
```

Comment:
Because it's called here
```
Component.onCompleted: {
    model.onSelectionChanged.connect(fretCanvas.selectionChanged)
}
```
So we can define a slot for `fretCanvas` when model's `selectionChanged` is emitted